nginx-custom.conf

```diff
@@ -0,0 +1,9 @@
+server {
+    listen 80;
+    server_name  localhost;
+    location / {
```

Comment:
What are these changes for?

Reply:
This has to do with the quality of life dockerfile upgrade.  Basically if you have a localhost link and you copy it and try to go to it in another tab, it will say it cannot resolve the host so you are forced to manually go through the site again to where the link takes you.  But this allows you to just go directly to the location via the url